### ISSUE TITLE
Mac Ios?

### ISSUE THREAD

**Issue by Antrosz01** [PocketMobsters/PokeMobBot#583]
Can be used on Mac iOS X?

**Comment by UraharaKisuke07** [comment]
afaik no. you can try simulating windows and then use this. I tried with wine but it doesn't work.

**Comment by Antrosz01** [comment]
Yes, I use Parallels Desktop, but maybe a change of code from iOS xD

**Comment by webbzac** [comment]
Also by the way iOS ≠ MacOS (OSX).
This bot is C# based run through an executable (.exe) so would not work on Mac OS(X).

**Comment by wynsmart** [comment]
Does anybody try to build on MacOS? I noticed C# now supports MacOS.

**Comment by webbzac** [comment]
@wynsmart Since when? The only way (afaik) to run .NET framework outside of a Windows Operating System is using something open-source like [Mono](http://www.mono-project.com/).

**Comment by wynsmart** [comment]
@webbzac Yep. So can we build with Mono? I didn't try this.

**Comment by webbzac** [comment]
I don't have a huge amount of experience in working with Mono but I'd start with this tool: http://mono-framework.com/MoMA

**Comment by GodLesZ** [comment]
I've _recently_ tested the N****Bot and had probems using the `SuperSocket` proxy. Mono crashed silently.
The website states also OS X support w/ Mono but at least the used lib seems to be not working with.
I'll give it a try using this version later.
